code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -46,7 +46,12 @@
 /obj/structure/closet/can_prevent_fall()
 	return TRUE
 
-/obj/structure/closet/Initialize()
+/obj/structure/closet/Initialize(mapload)
+	..()
+	populate_contents()
+	return mapload ? INITIALIZE_HINT_LATELOAD : INITIALIZE_HINT_NORMAL
```

Comment:
`populate_contents()` will be called twice if `mapload` is not null.

Reply:
welp, will fix

Reply:
done